ble/src/main/java/no/nordicsemi/android/ble/BleManagerHandler.java

```diff
@@ -838,7 +840,7 @@ private boolean internalSendNotification(@Nullable final BluetoothGattCharacteri
 		if (cccd == null)
 			return false;
 		// If notifications/indications were enabled, send the notification/indication.
-		final byte[] value = descriptorValues.containsKey(cccd) ? descriptorValues.get(cccd) : cccd.getValue();
+		final byte[] value = descriptorValues != null && descriptorValues.containsKey(cccd) ? descriptorValues.get(cccd) : cccd.getValue();
```

Comment:
No matter what, I think this null check is needed. `descriptorValues` is null checked elsewhere.